frontend/components/Card.tsx

```diff
@@ -12,7 +13,27 @@ const StyledCard = styled(AntdCard)`
   flex-direction: column;
   font-size: 0.8rem;
   padding: 1rem;
-  margin: 0.25rem;
+  width: 30%;
+
+  margin-right: min(3%, 39px);
+
+  @media ${maxDevice.laptop} {
+    flex-grow: 1;
+    width: 45%;
+    max-width: 100%;
+    margin: 0.25rem;
+    height: 138px;
+  }
+
+  @media (min-width: ${size.laptop + 1}px) {
+    &:nth-child(3) {
+      max-width: 41vh;
+      width: 30%;
+    }
+
+    max-width: 31vh;
+    padding: 2rem;
+  }
```

Comment:
Played around with the Card styling, here's what I like:


styled(AntdCard)`
   background-color: white;
   flex-direction: column;
   font-size: 0.8rem;
   padding: 1rem;

   margin-right: 3%;

   @media ${maxDevice.laptop} {
     flex-grow: 1;
     width: 45%;
     max-width: 100%;
     margin: 0.25rem;
  }
  @media (min-width: ${size.laptop + 1}px) {
     &:nth-child(3) {
       max-width: 35%;
       margin-right: 0;
     }

     max-width: 28%
     padding: 2rem;
   }